src/editor/editor.directive.ts

```diff
@@ -166,28 +167,49 @@ export class FroalaEditorDirective implements ControlValueAccessor {
     if (!this._opts.events) {
       this._opts.events = {};
     }
+
+    // Store user event callbacks if exist
+    if (this._opts.events[eventName]) {
+      this._userEventCallbacks[eventName] = this._opts.events[eventName];
+    }
+
     this._opts.events[eventName] = callback;
   }
 
   private initListeners() {
     let self = this;
 
     // bind contentChange and keyup event to froalaModel
-    this.registerEvent(this._element, "froalaEditor.contentChanged", function() {
+    this.registerEvent(this._element, 'contentChanged', function() {
       setTimeout(function() {
         self.updateModel();
+        if (self._userEventCallbacks['contentChanged'] && self._editor.events) {
+          self._editor.events.on('contentChanged', self._userEventCallbacks['contentChanged']);
```

Comment:
@akki744 this statement is wrong because it will bind event on every change. Which we don't want. I think Stefan mentioned to use self._editor.events.on('contnetChanged') instead of this.registerEvent().

Reply:
@navneetnagpal No this wont bind it on every callback and here we don't have editor instance initialized to bind events here. Its not possible to use editor.events here.